libs/sdk-backend-spi/src/workspace/dashboards/index.ts

```diff
@@ -106,6 +106,8 @@ export interface IGetDashboardOptions {
      * Defaults to false.
      */
     includeAvailableViaLink?: boolean;
+
+    exportId?: string;
```

Comment:
:-) yes

Reply:
Added this way, what would you think, is it ok or should we aim for more general timeout error (i.e. `TIMEOUT_ERROR` instead of `TIMEOUT_POLLING_ERROR`) 6f78923e93d97fb6a7e880267ae99a70bc6c44b9

Reply:
@pbenes I would also add note, that it has priority over filterContextRef? And its usage depends on the backend implementation, but I'm not sure how to properly formulate it. But we should note also this "contract"

Reply:
Tried something like efa43db7b5e7256bba40d3292dc085c41f09b586